engine/crates/parser-openapi/src/graph/debug.rs

```diff
@@ -0,0 +1,39 @@
+use std::fmt::Debug;
+
+use super::OpenApiGraph;
+
+/// std::fmt::Debug for a node in our graph
```

Comment:
Why not implement the normal debug trait? is it already implemented?

Reply:
The normal debug trait only has access to `&self`.  I'm implementing this trait for types that look like `struct OutputType(u32)` - so all the normal debug trait can print is `OutputType(123)`.  Which is not very useful for debugging things.

This trait has access to both `&self` and `&OpenApiGraph` - so it can look up the actual data that the `u32` inside `OutputType` refers to and print useful information.